client/src/main/scala/controller/AddTaskController.scala

```diff
@@ -0,0 +1,24 @@
+package controller
+
+import common.CodeEditor
+import org.scalajs.jquery._
+
+import scala.scalajs.js.JSApp
+
+object AddTaskController extends JSApp {
+  val buttonId = "submit"
+  var solutionTemplateEditor = new CodeEditor("solutionTemplate")
+  var referenceSolutionEditor = new CodeEditor("referenceSolution")
+  var suiteEditor = new CodeEditor("suite")
+
+  def main(): Unit = {
+    val submitButton = jQuery(s"#$buttonId")
+    submitButton.click(submit _)
+
+    def submit() = {
+      jQuery("#solutionTemplate").value(solutionTemplateEditor.value)
```

Comment:
copy contents from editors to normal fields so that standard POST request will send them to server and no changed for backend are needed.